docs/version0.49.md

```diff
@@ -28,6 +28,7 @@ The following new features and notable changes since version 0.48.0 are included
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
 - [Change to the shared classes cache generation number](#change-to-the-shared-classes-cache-generation-number)
 - [New shared classes cache suboption added to adjust the number of startup hints that can be stored](#new-shared-classes-cache-suboption-added-to-adjust-the-number-of-startup-hints-that-can-be-stored)
+- [`-Xgc` options for `subAllocator` heap size added to control the compressed reference allocation](#-xgc-options-for-suballocator-heap-size-added-to-control-the-compressed-reference-allocation)
```

Comment:
Pls remove "heap size", which can be replaced with "are"

Reply:
You meant "area"?

Reply:
No, I mean `are`, to produce the following, or rewrite it how you wish to remove `heap`
[`-Xgc` options for `subAllocator` are added to control the compressed reference allocation

Reply:
Is it correct to say "SubAllocator related `Xgc` options added to control the compressed reference allocation"?

Reply:
yes